docs/integrations/app-development/opentelemetry/jira.md

```diff
@@ -0,0 +1,225 @@
+    -   worklog changed (jira:worklog_updated)
+-   User related:
+    -   created (user_created)
+    -   updated (user_updated)
+    -   deleted (user_deleted)
+-   Sprint related:
+    -   created (sprint_created)
+    -   deleted (sprint_deleted)
+    -   updated (sprint_updated)
+    -   started (sprint_started)
+    -   closed (sprint_closed)
+
+When you configure the webhook, enter the URL for the HTTP source you created in [step 2](https://help.sumologic.com/docs/integrations/app-development/jira/#Step_2:_Configure_hosted_collector_to_receive_webhooks) as the endpoint for the webhook.
+
+## Logs Collection Configuration & App installation
+
+As part of the setting up the collection process and app installation user can select the App from App Catalog and click on **Install App**. Please follow the steps below:
+
+### Step 1: Set up Collector
+
+If you want to create a new Collector please select **Add a new Collector** option.
+
+:::note
+If you want to use an existing OpenTelemetry Collector, you can skip this step by selecting the **Use an existing Collector** option.
+:::
+
+Select the platform for which you want to install the Sumo OpenTelemetry Collector.
+
+This will generate a command which can be executed in the machine which needs to get monitored. Once executed it will install the Sumo Logic OpenTelemetry Collector agent.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Jira-OpenTelemetry/Jira-Collector.png')} alt="Collector" />
+
+### Step 2: Configure integration
+
+In this step we will be configuring the yaml file required for the Jira Collection.
+
+Path of the different log file configured to capture Jira logs is needed to be given here:
+
+-   Jira Access Logs. Apache HTTP server log files. Default path `/var/log/apache2/*.log`.
+-   Jira Security Logs. Security-related information. Default path `/home/jira/atlassian/application-data/jira/log/atlassian-jira-security.log`.
+-   Jira Catalina Logs. The application server log file. Default path `/home/jira/atlassian/application-data/jira/log/*.log`.
+
+Click on the **Download YAML File** button to get the yaml file.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Jira-OpenTelemetry/Jira-YAML.png')} alt="YAML" />
+
+### Step 3: Sending logs to Sumo
+
+Once you have the yaml file downloaded in step 2, follow the below steps based on the platform of the machine :
+
+Linux:
+
+1.  Copy the yaml file to `/etc/otelcol-sumo/conf.d/` folder in the Jira instance which needs to be monitored.
+2.  Place Env file in the following directory
+```sh
+/etc/otelcol-sumo/env/
+```
+3.  restart the collector using
+```sh
+ sudo systemctl restart otelcol-sumo
+```
+
+Windows:
+
+1.  Copy the yaml file to `C:\ProgramData\Sumo Logic\OpenTelemetry Collector\config\conf.d` folder in the machine which needs to be monitored.
+2.  Restart the collector using 
+```sh
+Restart-Service -Name OtelcolSumo
+```
+
+Mac
```

Comment:
```suggestion
Mac:
```

Reply:
@jpipkin1 This section needs to be put in a table, as it is in the other companion docs.